ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/util/EntityUtilTest.java

```diff
@@ -76,6 +77,27 @@ public void shouldBuildCorrectMapField() {
         equalTo("INTEGER"));
   }
 
+  @Test
+  public void shouldBuildCorrectDecimalField() {
+    // Given:
```

Comment:
nice! you picked up on our Given/When/Then testing pattern quickly 😂

Reply:
Big fan of bdd